gapic/src/main/com/google/gapid/perfetto/views/SliceSelectionView.java

```diff
@@ -77,6 +80,29 @@ public SliceSelectionView(Composite parent, State state, SliceTrack.Slice slice)
       createLabel(main, slice.name);
     }
 
+    RenderStageInfo renderStageInfo = slice.getRenderStageInfo();
+    if (renderStageInfo != null) {
+      LinkedHashMap<String, String> props = new LinkedHashMap<String, String>();
```

Comment:
A bit nitpicky, but LinkedHashMap seems unnecessary.
You could have done check to see if any handles exist first to call `withLayoutData` and then call `createLabel` directly with each handle.

Reply:
It'll be easier to move this if we want, if its stored as a map. Even if we decide to move to the properties pane, we need to know the number of elements that is going to be shown before creating the properties grid. Also, performance wise as well, there wont be any difference.